symplyphysics/laws/chemistry/lennard_jones_potential.py

```diff
@@ -0,0 +1,53 @@
+from sympy import Eq
```

Comment:
Don't use marketing names for laws.

Reply:
There are lots of other potentials that target the same topic (Mie, Buckingham, Morse potentials) though

Reply:
Maybe `lennard_jones_intermolecular_pair_potential`?

Reply:
Hm, good point. I suggest to add folder, eg `potential_energy_models` and put different models in it. This way we can use fancy model names without cluttering the project.

Reply:
I agree

Reply:
I think folder should have `__init__.py` to work properly. Tests don't need it.